specs/asgi.rst

```diff
@@ -217,6 +229,14 @@ where the ``protocol`` matches the scope type, and ``message_type`` is
 defined by the protocol spec. Examples of a message ``type`` value include
 ``http.request`` and ``websocket.send``.
 
+.. note::
+
+  Applications should actively reject any protocol that they do not understand
+  with an Exception (of any type). Failure to do this may result in the server
+  thinking you support a protocol you don't, which is especially dangerous with
+  the Lifespan protocol, as the server will wait to start until you tell it.
```

Comment:
Wondering if there might be an alternate to: "which is especially dangerous with the Lifespan protocol, as the server will wait to start until you tell it." - I think wording it as a *dangerous* case is maybe overly strong?

Reply:
True, it's not dangerous per se. I'll reword to say "confusing".